src/Compiler/Microsoft.NET.Sdk.Razor.SourceGenerators/RazorSourceGenerator.cs

```diff
@@ -321,13 +320,15 @@ public void Initialize(IncrementalGeneratorInitializationContext context)
                 context.AddSource(hintName, csharpDocument.GeneratedCode);
             });
 
-            context.RegisterHostOutput(processed(designTime: true), static (context, pair, _) =>
-            {
-                var (filePath, document) = pair;
-                var hintName = GetIdentifierFromPath(filePath);
-                context.AddOutput(hintName + ".rsg.cs", document.CodeDocument.GetCSharpDocument().GeneratedCode);
-                context.AddOutput(hintName + ".rsg.html", document.CodeDocument.GetHtmlDocument().GeneratedCode);
-            });
+            // ExternalAccess.RazorCompiler does not have IVT to the new assembly.
```

Comment:
Do we need to address this before merging?

Reply:
That needs the Roslyn counterpart first (https://github.com/dotnet/roslyn/pull/69655) but that seems better to be merged after this one (because it has a reference to the changed assembly name), so I think I will first merge this Razor PR, then the Roslyn PR, and then follow up by re-enabling this EA thing. It's currently not used anyway.